man/veritytab.xml

```diff
@@ -60,13 +60,61 @@ This is based on crypttab(5).
 
     <variablelist class='fstab-options'>
 
+      <varlistentry>
+        <term><option>no-superblock</option></term>
```

Comment:
i think it would be nicer to do superblock=BOOL

Reply:
Well, the reason why I picked up the negative form is to stick to the original tool `veritysetup` and to avoid confusion between both implementations of this utility in `systemd` and `cryptsetup`.

I guess they have chosen this form because the command `veritysetup format ....` creates a hash device with the superblock by default. If we want to save a block and remove the superblock we have to specify the superblock content with all the options when the block device is opened (`--no-superblock` implies `--hash`, `--format`,  `--data-block-size`,  `--hash-block-size`,  `--data-blocks`, `--hash-offset`, `--salt` and `--uuid`).

I do not have a strong opinion of switching to the *not-negative* form so if you really do prefer it, I can do it.

Reply:
ok, let's stick to the veritysetup names if they are defined, but otherwise let's please use normalized non-inverted settings

Reply:
will do `superblock=BOOL` in the end.

Reply:
done